tests/test_app.py

```diff
@@ -1,4 +1,61 @@
-def test_app_hello_world_success(sapinvoices_client):
-    response = sapinvoices_client.get("/")
+from flask import session
+from flask_login import current_user
+
+from webapp.app import User
+
+
+def test_app_request_index_success(
+    sapinvoices_client, mock_parse_oidc_data, mock_request_headers_oidc_data
+):
+    response = sapinvoices_client.get("/", headers=mock_request_headers_oidc_data)
     assert response.status_code == 200  # noqa: PLR2004
```

Comment:
Could we use `from http import HTTPStatus` and the associated enums like `HTTPStatus.OK` for these unit tests?

Reply:
Neato 😎 ! Updated and will use this for future tests.